text/0000-prelude-2021.md

```diff
@@ -0,0 +1,164 @@
+- Feature Name: `prelude_2021`
+- Start Date: 2021-02-16
+- RFC PR: [rust-lang/rfcs#0000](https://github.com/rust-lang/rfcs/pull/0000)
+- Rust Issue: [rust-lang/rust#0000](https://github.com/rust-lang/rust/issues/0000)
+
+# Summary
+[summary]: #summary
+
+A new prelude for the 2021 edition, featuring several extra traits.
+
+# Motivation
+[motivation]: #motivation
+
```

Comment:
I feel like a thing that's missing here is a general policy for _what_ we consider prelude-worthy. We don't have one right now, and my gut feeling is that this RFC goes many steps too far in expanding the prelude. Having an actual policy we can verify proposals against seems to be better than having a grab bag of things that people _think_ should be in the prelude.

Reply:
Yes, it would help future edition changes make choices about the prelude, even if it's just a ponce... a policy only used once, I mean. We should be later able to revisit and see if the rationales were Bad, Actually.

Reply:
👍 to this.  Without an agreed philosophy I suspect this will get bogged down substantially by all the details, since every single possible edition could be its own long conversation.

Operationally, to get it in for the edition, I think it would make sense to focus this to a small number of minimally-controversial changes that very clearly fit the philosophy and *need* to be edition tied.  Then future RFCs can be about specific additions to focus the conversation about them.

Reply:
>  I think it would make sense to focus this to a small number of minimally-controversial changes that very clearly fit the philosophy and need to be edition tied

Strong agree.

I'd imagine this goes along the lines of: 
 - Figure out a policy (disallow discussing individual items until we have one, perhaps revert to pre-rfc until we have one)
 - Figure out items which the policy clearly applies to
 - People can disagree with the policy or disagree that the policy applies to an included element.